.github/workflows/upload-and-deploy.yaml

```diff
@@ -257,7 +259,8 @@ jobs:
         if: matrix.tag_name == 'aws_glue_4'
         run: >
           su - glue_user --command "cd $GITHUB_WORKSPACE &&
-          python3 -m pytest tests/test_json_to_parquet.py --namespace $NAMESPACE -v"
+          python3 -m pytest tests/test_json_to_parquet.py --namespace $NAMESPACE -v &&
+          python3 -m pytest tests/test_run_great_expectations_on_parquet.py -v"
```

Comment:
ditto separate steps

Reply:
See comment above. Also particularly for this one, since we're using aws glue 4.0 for `test_run_great_expectations_on_parquet.py` for now, I will keep this under here.